client/landing/stepper/declarative-flow/new-hosted-site-flow.ts

```diff
@@ -72,6 +83,15 @@ const hosting: Flow = {
 			}
 
 			switch ( _currentStepSlug ) {
+				case 'domains': {
```

Comment:
Shouldn't we also add the domain to the cart? https://github.com/Automattic/wp-calypso/blob/07c71dc977b633a64a08cb3915ea44d92fb60450/client/landing/stepper/declarative-flow/onboarding.ts#L90

Reply:
Seems it's not required 🤔 

![CleanShot 2024-10-29 at 17 32 14](https://github.com/user-attachments/assets/cef9031d-8bc6-49b7-9aed-a936a1651007)

Reply:
I have tested it, and the domain is added to the cart, I think that is done by [this handler](https://github.com/Automattic/wp-calypso/blob/e26e2c188c2eefc5dec57015d63a8a1461ffd37f/client/landing/stepper/declarative-flow/internals/steps-repository/domains/index.tsx#L269) that eventually sets [the domain cart item](https://github.com/Automattic/wp-calypso/blob/e26e2c188c2eefc5dec57015d63a8a1461ffd37f/client/landing/stepper/declarative-flow/internals/steps-repository/domains/index.tsx#L125)